### DISCUSSION TITLE
Keymap context or keymaps for project search/diagnostics/references buffers?

### DISCUSSION THREAD

**Discussion by liouk** [zed-industries/zed#15962] [Feature Requests]
Is there a way to define keymaps specifically for buffers like project search/diagnostics/references? (e.g. move to the next/previous match/diagnostic/reference)

Currently their keymaps seem to be defined simply in the `Editor` context (or `VimControl`, when vim mode is enabled).

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
